packages/workbox-cache-expiration/Plugin.mjs

```diff
@@ -235,7 +242,7 @@ class Plugin {
    * There is no Workbox-specific method needed for cleanup in that case.
    */
   async deleteCacheAndMetadata() {
-    // Do this one at at a time instance of all at once via `Promise.all()` to
+    // Do this one at at a time instead of all at once via `Promise.all()` to
```

Comment:
While we're fixing grammar, remove the duplicate "at" earlier in this line.

Reply:
Done.